### PR TITLE
add l10n Authentication feature

### PR DESCRIPTION
**Pull request by tigershi**

### CODE REVIEW

File: g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/conf/SwaggerConfig.java [hunk 1]
```diff
@@ -89,7 +89,7 @@
 	@ConditionalOnProperty(value = "vipservice.authority.enable")
```
Comment (Xiaochao8) [inline g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/conf/SwaggerConfig.java:89]: Should change this to authentication?

**Review by Xiaochao8 — DISMISSED** [review]

**Review by jin-jasper — DISMISSED** [review]

File: g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/conf/SwaggerConfig.java (hunk 1 above)
Comment (tigershi) [reply g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/conf/SwaggerConfig.java:89]: this just change it in swaggerUI.  this use in configuration

File: g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/crons/SourceSendingCron.java [hunk 2]
```diff
@@ -72,8 +72,21 @@
...
+	
+	@Value("${vip.server.authentication.token:#}")
+	private String remoteVIPAuthAppToken;
+	
 	@Value("${spring.profiles.active}")
 	private String activeDaoType;
+	
+	
+	
+
```
Comment (linr211) [inline g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/crons/SourceSendingCron.java:89]: remove the empty lines.

File: g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/crons/SourceSendingCron.java [hunk 3]
```diff
@@ -265,9 +278,22 @@ private void putDataToRemoteVIP(ComponentSourceDTO cachedComDTO) throws VIPHttpE
...
+			HTTPRequester.putJSONStr(jsonStr, urlStr, header);
+		
 	}
 
+	
+	
+	
+	
+	
+
```
Comment (linr211) [inline g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/crons/SourceSendingCron.java:296]: remove the empty lines.

File: g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/crons/SourceSendingCron.java (hunk 2 above)
Comment (tigershi) [reply g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/crons/SourceSendingCron.java:89]: done

File: g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/crons/SourceSendingCron.java (hunk 3 above)
Comment (tigershi) [reply g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/crons/SourceSendingCron.java:296]: done

**Review by Xiaochao8 — APPROVED** [review]

**Review by jin-jasper — APPROVED** [review]

**Pull request merged**